source/tutorial/enforce-keyfile-access-control-in-existing-sharded-cluster-no-downtime.txt

```diff
@@ -0,0 +1,297 @@
+
+Restart Each ``mongos`` Instance without ``transitionToAuth``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. important::
+
+  At the end of this section, clients must specify authentication
+  credentials to connect to the sharded cluster. Update clients to specify
+  authentication credentials *before* completing this section to avoid loss of
+  connectivity.
+
+To complete the transition to fully enforcing authentication in the
+sharded cluster, you must restart each :program:`mongos` instance without
+the :setting:`security.transitionToAuth` setting.
+
+Perform these steps for each :program:`mongos` in the sharded cluster,
+working on one :program:`mongos` at a time.
+
+.. include:: /includes/steps/enable-authentication-in-shardcluster-nodowntime-auth-mongos.rst
+
+At the end of this section, all :program:`mongos` instances enforce client
+authentication and :setting:`security.keyFile` internal authentication.
+
+Restart Each Config Server Replica Set Member without ``transitionToAuth``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. important::
+
+  At the end of this step, clients must specify authentication credentials to
+  connect to the config server replica set. Update clients to specify
+  authentication credentials *before* completing this section to avoid loss of
+  connectivity.
+
+To complete the transition to fully enforcing authentication in the
+sharded cluster, you must restart each :program:`mongod` instance without
+the :setting:`security.transitionToAuth` setting.
+
+Perform these steps for each :program:`mongod` in the config server replica
+set, one member at a time. Work on the :term:`secondary` or
+:term:`arbiter` members *first*, and the :term:`primary` *last*.
+
+.. include:: /includes/steps/enable-authentication-in-shardcluster-nodowntime-auth-config.rst
+
+At the end of this section, all :program:`mongod` instances in the config
+server replica set enforce client authentication and
+:setting:`security.keyFile` internal authentication.
+
+Restart Each Member in Each Shard Replica Set without ``transitionToAuth``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. important::
+
+  At the end of this step, clients must specify authentication credentials to
+  connect to the shard replica set. Update clients to specify authentication
+  credentials *before* completing this section to avoid loss of connectivity.
+
+To complete the transition to fully enforcing authentication in the sharded
+cluster, you must restart every member of every shard replica set in the
+sharded cluster without the :setting:`security.transitionToAuth` setting.
+
+Repeat these steps for each shard replica set in the sharded cluster.
+
+Perform these steps on one shard replica set at a time, and one member in the
+shard replica set at a time. Work on the :term:`secondary` or :term:`arbiter`
+members *first*, and the :term:`primary` *last*.
```

Comment:
Unless explicitly requested to mention the arbiters, I might remove the arbiter members.
It sounds like you can have multiple arbiters.